scripts/tokenomics-emulation/config.json

```diff
@@ -1,5 +1,15 @@
 {
   "network": {
+    "hosts": [
+      {
```

Comment:
Here will be all hosts including genesis. Please, add placeholders.

Reply:
added